packages/jest-util/src/index.js

```diff
@@ -31,11 +33,26 @@ const createDirectory = (path: string) => {
   }
 };
 
+const clearDirectory = (directoryPath: string) => {
```

Comment:
Why not just `rimraf`? (or `del` or one other of the many modules that does this)

Reply:
Only because I don't generally add dependencies to someone else's project unless they mention it. `rimraf` is fine.

Reply:
Migrated to rimraf.